### ISSUE TITLE
renderAsLayer property should be exposed to public somehow

### ISSUE THREAD

**Issue by garnett** [dblock/ios-snapshot-test-case-expecta#24]
There as some problems with hardcoded ```renderAsLayer=YES``` on [iOS 8](https://github.com/facebook/ios-snapshot-test-case/issues/52)
My idea is to have separate set of matchers matching views which are not rendered as layer but as view hierarchy, because I am almost sure there still will be cases where one of the rendering modes is more suitable.
If you think this change can fit the library design I can make a PR

**Comment by orta** [comment]
This is a cool idea, I like the implementation but that this point we should start breaking the implementation out into something more abstract, a 500 line file filled with methods gets hard to follow.

**Comment by foulkesjohn** [comment]
I agree, I just needed the feature quickly and haven't had chance to revisit it yet.

**Comment by orta** [comment]
This function has been removed from the snapshots lib, so closing the issue as there's nothing to expose.
